doc/intro.rst

```diff
@@ -176,15 +176,15 @@ storing test log in the Test object, passed JSON file can look as follows:
 
 Both forms are supported. In case log entry is missing or simple JSON
 format is used, logs for each Test object are empty. They can be filled
-in using plugins.
+in using plugins [at later time??].
```

Comment:
@pfalcon yes! After results are fetched, [enabled plugins](https://github.com/Linaro/squadplugins) will post-process tests seeking for extra logs. One example is a built-in plugin `linux-log-parser`, which searches job log for linux kernel oopsies, then create tests for each specific type of oops (kernel-panic, warning, bug, etc)

Reply:
I propose to change it like this:
"They can be filled in using plugins in a post-processing step. This happens after all results are collected in SQUAD"